cmd/protoc-gen-goclay/genhandler/tpl_servicedesc.go

```diff
@@ -81,6 +81,11 @@ func (d *{{ $svc.GetName | goTypeName }}Desc) RegisterHTTP(mux {{ pkg "transport
             return
         }
 
+        if ctxErr := r.Context().Err(); ctxErr != nil && ctxErr == context.Canceled {
+           w.WriteHeader(499) // Client Closed Request
```

Comment:
FYI, it's just nginx invention, not a standard code:
https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml

But it does not matter for the client anyway :)

Reply:
I know it, that's why here is the int value, but not a status const from http package :)

And yes you are right, it does not matter for the client because it already gone away, but it make sense for metrics.